frontend/package.json

```diff
@@ -17,14 +17,17 @@
     "@hookform/error-message": "2.0.0",
     "@hookform/resolvers": "2.9.6",
     "@reduxjs/toolkit": "1.8.3",
+    "@types/react-table": "7.7.12",
     "clsx": "1.2.1",
     "guruhub-shared": "file:../shared/build",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "7.33.1",
     "react-redux": "8.0.2",
     "react-router-dom": "6.3.0",
-    "react-scripts": "5.0.1"
+    "react-scripts": "5.0.1",
+    "react-table": "7.8.0",
+    "save-dev": "0.0.1-security"
```

Comment:
I think save-dev package shouldn't be installed, probably you ran `npm i save-dev` instead of `npm i --save-dev <package-name>`